.gitmodules

```diff
@@ -19,3 +19,7 @@
 [submodule "litex/deps/litedram"]
 	path = litex/deps/litedram
 	url = https://github.com/enjoy-digital/litedram.git
+[submodule "migen/deps/migen"]
+	path = migen/deps/migen
+	url = https://github.com/m-labs/migen.git
+	branch = 0.6.dev-283-g562c046
```

Comment:
I don't think we need a `branch = XXXX` as submodules are always linked to a specific commit anyway?